CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This minor release:
 
   A backward compatibility module has been created `io.jsonwebtoken:jjwt-deprecated`, if you are compiling against 
   these classes directly, otherwise you will be unaffected.
+* Adds support for custom types when deserializing with Jackson. To use configure your parser with `Jwts.parserBuilder().deserializeJsonWith(new JacksonDeserializer(Maps.of("claimName", YourType.class))).build()`.
```

Comment:
Are we going to create the `Maps.of` utility method? (would reside, presumably, in `io.jsonwebtoken.lang.Maps`).

Reply:
Yeah, easy enough to add. I was thinking of recommending just using the Java 11 `Map.of`, but I know many of us will be on 8 for a while yet.